sys/windows/zsyscall_windows.go

```diff
@@ -2,40 +2,71 @@
 
 package windows
 
-import "unsafe"
-import "syscall"
+import (
+	"syscall"
+	"unsafe"
+)
 
 var _ unsafe.Pointer
 
+// Do the interface allocations only once for common
+// Errno values.
+const (
+	errnoERROR_IO_PENDING = 997
+)
+
+var (
+	errERROR_IO_PENDING error = syscall.Errno(errnoERROR_IO_PENDING)
+)
+
+// errnoErr returns common boxed Errno values, to prevent
+// allocations at runtime.
+func errnoErr(e syscall.Errno) error {
+	switch e {
+	case 0:
+		return nil
+	case errnoERROR_IO_PENDING:
+		return errERROR_IO_PENDING
+	}
+	// TODO: add more here, after collecting data on the common
+	// error values see on Windows. (perhaps when running
+	// all.bat?)
+	return e
+}
+
 var (
 	modkernel32 = syscall.NewLazyDLL("kernel32.dll")
 	modpsapi    = syscall.NewLazyDLL("psapi.dll")
 	modntdll    = syscall.NewLazyDLL("ntdll.dll")
 	modadvapi32 = syscall.NewLazyDLL("advapi32.dll")
```

Comment:
@andrewkroh I had to revert these lines by hand, recent go versions use `windows.NewLazySystemDLL`, but as this module is named `windows` it fails.

We should move this syscalls to other submodule, or rename this one.

Reply:
What fails? It should be fine to import `golang.org/x/sys/windows` from within `package windows`. That might be a new dep that we didn't use previously though.

Reply:
The problem is that it generates this code here:
```
       modkernel32 = NewLazySystemDLL("kernel32.dll")
       modpsapi    = NewLazySystemDLL("psapi.dll")
       modntdll    = NewLazySystemDLL("ntdll.dll")
       modadvapi32 = NewLazySystemDLL("advapi32.dll")
```

It seems that because it has some special handling for packages named `windows`: https://github.com/golang/go/blob/go1.10.5/src/syscall/mksyscall_windows.go#L735

I have seen that adding `-systemdll=false` to the parameters of `mksyscall_windows` generates the same code as before. I am adding it by now.